client/electron/go_vpn_tunnel.ts

```diff
@@ -47,6 +47,7 @@ const DNS_RESOLVERS = ['1.1.1.1', '9.9.9.9'];
 // about the others.
 export class GoVpnTunnel implements VpnTunnel {
   private readonly tun2socks: GoTun2socks;
+  private readonly connectivityChecker: GoTun2socks;
```

Comment:
Why is this type `GoTun2socks`? Why do we need two?

Reply:
Because these two processes might run at the same time. For example, we will [check connectivity here](https://github.com/Jigsaw-Code/outline-apps/blob/master/client/electron/go_vpn_tunnel.ts#L151), and tun2socks is still running [until we stop it later](https://github.com/Jigsaw-Code/outline-apps/blob/master/client/electron/go_vpn_tunnel.ts#L163).